components/cam/src/physics/cam/micro_p3.F90

```diff
@@ -674,6 +676,12 @@ SUBROUTINE p3_main(qc,nc,qr,nr,th_old,th,qv_old,qv,dt,qitot,qirim,nitot,birim,ss
              birim(i,k) = 0.
           endif
 
+
+         !Activaiton of cloud droplets 
+          if (log_predictNc) then 
+             nc(i,k) = nc(i,k) + npccn(i,k) * dt
+          endif 
+
```

Comment:
This addition isn't in the original PR #72 , does this reflect one of the newer changes requested by Peter C?

Reply:
That is correct, this was in response to Peter C comment  on PR #72 about where droplet nucleation should occur. I should have made that more clear.